src/vs/editor/contrib/clipboard/browser/clipboard.ts

```diff
@@ -301,7 +302,8 @@ if (PasteAction) {
 
 	// 2. Paste: (default) handle case when focus is somewhere else.
 	PasteAction.addImplementation(0, 'generic-dom', (accessor: ServicesAccessor, args: any) => {
-		const triggerPaste = accessor.get(IClipboardService).triggerPaste();
+		const targetWindowId = getWindowId(getActiveWindow());
+		const triggerPaste = accessor.get(IClipboardService).triggerPaste(targetWindowId);
```

Comment:
You could do `getActiveWindow().vscodeWindowId`

Reply:
Cool thanks for the review, will do!